packages/components/src/components/TextInput/TextInput.tsx

```diff
@@ -42,16 +43,20 @@ type TextInputProps = {
 };
 
 const TextFieldRoot = forwardRef<HTMLInputElement, TextInputProps>(
-    ({ children, className, status = 'neutral', 'data-test-id': dataTestId, ...inputProps }, ref) => {
+    (
+        { children, className, status = 'neutral', 'data-test-id': dataTestId = 'fondue-text-input', ...inputProps },
+        ref,
+    ) => {
         return (
             <div
                 className={cn(rootStyles, className)}
                 data-disabled={inputProps.disabled}
                 data-read-only={inputProps.readOnly}
                 data-status={status}
-                data-test-id={dataTestId ?? 'fondue-text-input'}
+                data-test-id={dataTestId}
             >
-                <div className={loadingStatusStyles} data-test-id="fondue-text-input-loader" />
+                {status === 'loading' ? <div className={loadingStatusStyles} data-test-id="loader" /> : null}
```

Comment:
wouldn't it make sense to use `${dataTestId}-loader`?

Reply:
Internal use only, external usage shouldn't know about this test id.